src/main/java/net/atos/zac/zoeken/converter/ZaakZoekObjectConverter.java

```diff
@@ -162,7 +162,7 @@ private void addBetrokkenen(final Zaak zaak, final ZaakZoekObject zaakZoekObject
     }
 
     private User findBehandelaar(final Zaak zaak) {
-        return zgwApiService.findBehandelaarForZaak(zaak)
+        return zgwApiService.findBehandelaarMedewerkerRoleForZaak(zaak)
                 .map(behandelaar -> identityService.readUser(
```

Comment:
Strange pattern used here. I'd 've thought that you should map the `behandelaar` to `.getBetrokkeneIdentificatie().getIdentificatie()` and use that to `identityService.readUser`... or is there a flaw in that reasoning?

Reply:
sorry, I am not going to even look at this code since it has nothing to do with this (already very large) PR.. Sure, a lot of existing code could be improved but before touching any of that I would first want to make sure we have adequate test coverage. Not in this PR.